openmdao/recorders/tests/test_sqlite_reader.py

```diff
@@ -1255,6 +1255,7 @@ def test_list_inputs_with_includes_excludes(self):
     def test_list_discrete(self):
         model = om.Group()
 
+        # TODO: Change x to an auto_ivc variable.
         indep = model.add_subsystem('indep', om.IndepVarComp())
```

Comment:
Is there a bug in the case stuff here?

Reply:
I thought we discussed that discrete's don't work with auto_ivc yesterday.  Will test it.

Reply:
Oh sorry, I didn't notice this was discrete.  Yeah, if it doesn't work, put the TODO back. I've got drivers next, so I will fix any discrete problems.

Reply:
I have a fix up for this.  You can probably pull it down and fix this test.